cmd/soroban-cli/src/print.rs

```diff
@@ -86,6 +86,8 @@ create_print_functions!(error, errorln, "❌");
 create_print_functions!(globe, globeln, "🌎");
 create_print_functions!(info, infoln, "ℹ️");
 create_print_functions!(link, linkln, "🔗");
+create_print_functions!(over, overln, "🔄");
```

Comment:
I'm using this specifically when we're overwriting a file in init. I'd love some other ideas for what to call this!

Reply:
`over` or `overwrite` sounds fine

Reply:
+1 for `overwrite`.

Reply:
The icon doesn't speak overwrite to me though. The icon feels like "swap".

What if for overwrites we wrote two lines, in much the same way git might communicate this. E.g.:
```
➕ Writing file1...
➕ Writing file2...
➖ Removing existing file3...
➕ Writing file3...
```

It might also be reasonable to use the same icon
```
➕ Writing file1...
➕ Writing file2...
➕ Writing file3 (overwriting existing file)...
```

Reply:
I like @leighmcculloch's one-liner suggestion

```
➕ Writing file3 (overwriting existing file)...
```

Reply:
I like that suggestion!